packages/ui/src/components/va-checkbox/VaCheckbox.demo.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <VbDemo>
+  <VbDemo class="va-input-demo">
```

Comment:
```suggestion
  <VbDemo class="d-flex direction--column">
```

But, do we actually need it if checkbox is block?

Reply:
I've decided not delete it - we will need it after we make checkbox inline.